### PR TITLE
General-Application: suggest more details in applications

### PR DESCRIPTION
**Pull request by chriscool**

### CODE REVIEW

File: General-Application-Information.md [hunk 1]
```diff
@@ -57,9 +57,14 @@ documentation on their web site that you should follow.
...
 your microproject in your application (e.g. linking to
-[lore.kernel.org](https://lore.kernel.org/git/)). If you
-participate in the review of other patches, then you may also include
-links to discussions that would support your application.
+[lore.kernel.org](https://lore.kernel.org/git/)). If you participate
+in the review of other patches, then you may also include links to
+discussions that would support your application. Please also describe
+the status of the patches you have sent and maybe reviewed: Have they
+been merged already in an official branch maintained by Junio? What
+does Junio's "What's cooking in git.git" emails say about it? In
```
Comment (sivaraam) [inline General-Application-Information.md:65]: Given that we're very likely talking to new contributors here, I wonder if its worth linking to a resource that describes "What's cooking" e-mails briefly. 🤔 

So far, I could only find a small note in README: https://github.com/git/git/blob/master/README.md#L44-L47

**Review by sivaraam — APPROVED** [review]
Overall, this looks good. I just have one suggestion.

File: General-Application-Information.md (hunk 1 above)
Comment (chriscool) [reply General-Application-Information.md:65]: If there is no good description maybe we could just link to https://lore.kernel.org/git/?q=%22What%27s+cooking+in+git.git%22 ?

Or we could add a short description and this link to https://git.github.io/Hacking-Git/ ?

Comment (sivaraam) [reply General-Application-Information.md:65]: > 
> 
> If there is no good description maybe we could just link to [lore.kernel.org/git/?q=%22What%27s+cooking+in+git.git%22](https://lore.kernel.org/git/?q=%22What%27s+cooking+in+git.git%22) ?
> 

We could actually try the following link to reduce the noise: https://lore.kernel.org/git/?q=s%3A%22What%27s+cooking+in+git.git%22

> Or we could add a short description and this link to [git.github.io/Hacking-Git](https://git.github.io/Hacking-Git/) ?

Yeah. Mentioning in Hacking-Git might make it useful to a wider audience.

Comment (chriscool) [reply General-Application-Information.md:65]: Yeah, thanks for the improved link. I added it to Hacking-Git in d91fc3f841097ee4033a561d4949b9e2540c16d1.

Comment (chriscool) [reply General-Application-Information.md]: Further changes related to this are in cecd902c7f9e1c09f8a0a39b180e4d6daa5c4944 and 79238a64ce760eb060442a848cfcb36b99bb7411.

**Comment by chriscool** [comment]
Let's merge this now. We can always improve on it later.

**Pull request merged**